soh/soh/Enhancements/randomizer/randomizer.h

```diff
@@ -54,16 +55,16 @@ class Randomizer {
     bool IsTrialRequired(RandomizerInf trial);
     u8 GetRandoSettingValue(RandomizerSettingKey randoSettingKey);
     RandomizerCheck GetCheckFromActor(s16 actorId, s16 sceneNum, s16 actorParams);
-    RandomizerGet GetRandomizerGetFromActor(s16 actorId, s16 sceneNum, s16 actorParams);
-    RandomizerGet GetRandomizerGetFromKnownCheck(RandomizerCheck randomizerCheck);
+    RandomizerGetData GetRandomizerGetFromActor(s16 actorId, s16 sceneNum, s16 actorParams);
+    RandomizerGetData GetRandomizerGetFromKnownCheck(RandomizerCheck randomizerCheck);
```

Comment:
Sounds weird but more accurate
```suggestion
    RandomizerGetData GetRandomizerGetDataFromActor(s16 actorId, s16 sceneNum, s16 actorParams);
    RandomizerGetData GetRandomizerGetDataFromKnownCheck(RandomizerCheck randomizerCheck);
```

Reply:
I think I didn't rename those so as not to have to update where they are called, but VSCode should take care of most of that for me anyway I think.